test/fixtures/schema/array-with-nested-type.json

```diff
@@ -157,7 +157,7 @@
                                   "content": "application/json"
                                 }
                               },
-                              "content": "{\n  \"address\": [\n    4,\n    2,\n    42\n  ]\n}"
+                              "content": "{\n  \"address\": [\n    0,\n    4,\n    2,\n    42\n  ]\n}"
```

Comment:
This `0` is being introduced, similar to another comment I made about array type-hints above.